otx/algorithms/action/adapters/mmaction/models/recognizers/recognizer3d.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import functools
+from collections import OrderedDict
+from mmaction.models.builder import RECOGNIZERS
+from mmaction.models.recognizers.recognizer3d import Recognizer3D
+
+
+@RECOGNIZERS.register_module(force=True)
+class Recognizer3D(Recognizer3D):
+    """3D recognizer model framework for OTX compatibility"""
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        # Hooks for redirect state_dict load/save
+        self._register_state_dict_hook(self.state_dict_hook)
```

Comment:
does this function exist?
is it just I miss or not exist yet?

Reply:
Exists, github is sometimes slower than commit :(